### DISCUSSION TITLE
[Feature] Ability to restrict people search to just show exact people photos

### DISCUSSION THREAD

**Discussion by imohitkr** [immich-app/immich#12091] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When I went on a trip with my friends, I took a lot of photos. While I want to keep the group photos, I don't want to keep the individual photos or group photos where I do not happen to be present. It would be helpful to have a feature that allows me to search for and view only photos that exactly match the people's selection, not less or more. This way, the search results wouldn't display photos with more or fewer people than I specified. And we can bulk delete them. 

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#10373
